Makefile

```diff
@@ -93,6 +95,9 @@ bundle:
 	rm -rf dist/
 	mkdir -p dist/$(PLUGIN_ID)
 	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
+ifneq ($(ASSETS),)
```

Comment:
At first I thought this was a formalized proposal for including assets fetchable via HTTP (e.g. without any explicit plugin handling), but I believe that would require dumping things into `webapp/dist/assets` as well. This actually seems more interesting than just including static files into the build for use by the server, though I can see a case for both. (Though, sometimes it's easier to just embed the static files into the binary via something like https://github.com/rakyll/statik.)

Thoughts?

Reply:
This discussion lead to the creation of https://github.com/mattermost/mattermost-server/issues/10448.